neo/d3xp/Game_local.cpp

```diff
@@ -309,6 +310,10 @@ void idGameLocal::Init( void ) {
 	const idDict *dict;
 	idAAS *aas;
 
+	msec = 16.0; //60fps
```

Comment:
please put a space after "//", e.g. `// 60fps` (yes, always, not just here), as it's done in the existing code